### PR TITLE
gzll: nRF5340 network core support

### PR DESCRIPTION
**Pull request by leewkb4567**
Gazell Link Layer support for nRF5340 network core.

### CODE REVIEW

File: gzll/CHANGELOG.rst [hunk 1]
```diff
@@ -9,6 +9,17 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+Main branch
```
Comment (peknis) [inline gzll/CHANGELOG.rst:12]: Not so sure whether this should be something else, like nRF Connect SDK v2.1.0, for example.

Comment (leewkb4567) [reply gzll/CHANGELOG.rst:12]: Probably not. IMO, it will take a push for v2.2.0.

**Review by peknis — APPROVED** [review]

**Comment by leewkb4567** [comment]
Rebase to current main.

**Comment by leewkb4567** [comment]
Rebase to current main.

**Review by carlescufi — APPROVED** [review]

**Pull request merged**